docs/_docs/thin-clients/dotnet-thin-client.adoc

```diff
@@ -235,6 +235,20 @@ task named `org.foo.bar.AddOneTask` passing `1` as an input parameter:
 include::{sourceCodeFile}[tag=client-compute-task,indent=0]
 -------------------------------------------------------------------------------
 
+== Executing Ignite Services
+
+You can use the `IServicesClient` API to invoke an link:services/services[Ignite Service] that
+is *already deployed* in the cluster.
```

Comment:
Can we cross-link to a page/section explaining how the services can be deployed? Developers that start with thin clients might not be aware of that.

Reply:
Done